### PR TITLE
add feedstock for ixmp

### PR DESCRIPTION
**Pull request by gidden**

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,80 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml]: Please do not delete the example recipe.

File: recipes/ixmp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/iiasa/ixmp/archive/v{{ version }}.tar.gz
+  sha256: 512c63bc8473e667ba8571fb7125540ce47f5245b3cca71c989f089600cfd494
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (synapticarbors) [inline recipes/ixmp/meta.yaml]: It looks like there is a python entry point. Please include it:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

and add tests for the CLI:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

File: recipes/ixmp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - nbformat
+    - jupyter
+
+test:
+  imports:
+    - ixmp
+
+about:
+  home: https://github.com/iiasa/ixmp
+  license: Apache
```
Comment (synapticarbors) [inline recipes/ixmp/meta.yaml]: `license: Apache-2.0`

File: recipes/ixmp/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  build:
+    - python
+    - pip
+  run:
+    - openjdk
+    - jpype1 >=0.6.2
+    - cython
+    - numpy
+    - pandas
+    - pytest >=3.0.6
```
Comment (synapticarbors) [inline recipes/ixmp/meta.yaml]: Is `pytest` a `run` dependency, or is it only really required for testing. If it's the former, please remove it.

**Comment by gidden** [comment]
Hi @synapticarbors thanks for the review, I believe I've addressed all your comments.

**Pull request merged**